.github/workflows/test.yml

```diff
@@ -84,13 +84,6 @@ jobs:
           python-version: 3.9
       - name: Install tox
         run: pip install -U tox
-      - name: Prep mssql driver
-        if: ${{ matrix.tox-environment == 'docker-tests' }}
-        run: | 
-          curl https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
-          sudo sh -c "echo 'deb [arch=amd64,armhf,arm64] https://packages.microsoft.com/ubuntu/20.04/prod focal main' >  /etc/apt/sources.list.d/mssql-release.list"
-          sudo apt-get update
-          sudo ACCEPT_EULA=Y apt-get install -y msodbcsql17
```

Comment:
These changes are not really required. These were needed to fix related issues on an Ubuntu VM but the image we are using in CI doesn't need this.

Changes from #726